deb/Makefile

```diff
@@ -16,13 +16,15 @@ COMMON_FILES=common
 BUILD?=docker build \
 	--build-arg GO_IMAGE=$(GO_IMAGE) \
 	--build-arg COMMON_FILES=$(COMMON_FILES) \
+	--build-arg DEFAULT_PRODUCT_LICENSE="$(DEFAULT_PRODUCT_LICENSE)" \
```

Comment:
Let's just make a `PRODUCT_LICENSE` variable that we can override and have a default value set in all of the Makefiles.

Reply:
So you'd like a:
```make
DEFAULT_PRODUCT_LICENSE?="Community Engine"
```
in every makefile instead of just the top-level makefile?

Reply:
I've made the suggested change.